app/src/test/java/com/imaginaryrhombus/proctimer/TimerTestRunner.kt

```diff
@@ -0,0 +1,11 @@
+package com.imaginaryrhombus.proctimer
+
+import android.app.Application
+import android.content.Context
+import androidx.test.runner.AndroidJUnitRunner
+
+class TimerTestRunner: AndroidJUnitRunner() {
+    override fun newApplication(cl: ClassLoader?, className: String?, context: Context?): Application {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/fizxive/ProcTimer/blob/c47c64a9552d61bc5bc24e0f11a73730cdbc9776/app/src/test/java/com/imaginaryrhombus/proctimer/TimerTestRunner.kt#L8"></span>Exceeded max line length (100) (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>